docs/api.md

```diff
@@ -367,6 +367,33 @@ documented in the [Browser API ⇗](/docs/browser.md) documentation.
 
 * See [Browser API ⇗](/docs/browser.md)
 
+#### `transport` (Object)
+
+The `transport` option is a shortcut to the [pino.transport()](#pino-transport) function.
+It supports the same input options:
+```js
+require('pino')({
```

Comment:
do we do this require('pino') inline thing elsewhere in the docs?

Reply:
There are 18 occurrences of `require('pino')` in this file actually: would you like to migrate the examples to `pino`?

Reply:
Do they assign the result of `require` or do they use it directly like this example?

Reply:
Both declaration tbh

https://github.com/pinojs/pino/blob/5333d73773f59331c2d5c7962dec3ee00574fc6f/docs/api.md#destination-sonicboom--writablestream--string--object

I can update this example and the other ones if you tell me what do you prefer.
Is the cjs syntax not up to date for users?